src/pages/SearchTestPage.tsx

```diff
@@ -1,19 +1,46 @@
-import { useRef, useState } from "react";
+import { FunctionComponent, useRef, useState } from "react";
 import { useQuery } from "react-query"
 import { Link } from "react-router-dom";
 import useChainId from "../hooks/useChainId";
-import { searchMorphs } from "../lib/search";
+import { getLoaders } from "../lib/dataloaders";
+import { searchMorphsV2 } from "../lib/search-v2";
+
+const Address: FunctionComponent<{ address: string }> = ({ address }) => {
```

Comment:
ezpz ENS resolution in naive child component w/ galaxy-brain batching and de-duping via dataloader